std_msgs/Header header
int32[19] inputs